### DISCUSSION TITLE
Please add Subcene for subtitles is a very great site

### DISCUSSION THREAD

**Discussion by hishamaltmimy** [community/community#27785] [Accessibility]
Please add Subcene for subtitles is a very great site

**Comment by jfullstackdev** [comment]
what will be the use of it for GitHub as a feature ?

**Comment by benjiallen** [comment]
Hi @hishamaltmimy, can you give me more information about this idea? What does Subcene do and what value do you get out of it? How do you expect it to be integrated into GitHub services?
